src/util/internalCallTree.js

```diff
@@ -116,14 +116,16 @@ async function buildTree (tree, step, scopeId) {
     return false
   }
 
-  var currentSourceLocation = {}
+  var currentSourceLocation = {start: -1, length: -1, file: -1}
   while (step < tree.traceManager.trace.length) {
     var sourceLocation
     try {
       sourceLocation = await tree.extractSourceLocation(step)
-      if (sourceLocation.start !== currentSourceLocation.start ||
-      sourceLocation.length !== currentSourceLocation.length ||
-      sourceLocation.file !== currentSourceLocation.file) {
+
+      if (sourceLocation.file !== currentSourceLocation.file ||
```

Comment:
Can you write a helper for that? Something with a good name like `included`. Please also handle the special values of `-1`.